packages/compass/src/main/auto-updater.ts

```diff
@@ -26,7 +29,29 @@ class LinuxAutoUpdater extends EventEmitter implements AutoUpdater {
   }
 }
 
-const autoUpdater =
-  process.platform === 'linux' ? new LinuxAutoUpdater() : electronAutoUpdater;
+function hasSquirrel() {
+  const updateExe = path.resolve(
```

Comment:
random question .. this seems to be sort-of safe at least for now, but also a bit scary that we are relying on some squirrel internal detail. I don't know how likely it would be, but if this condition would change at some point without us noticing then we would break all the windows auto-updates .. and the lack of automated test makes me a bit nervous.

Is there a chance we could rely on something else .. like the electronUpdater directly or something else that would directly break? Or maybe there is a way to actually test it